packages/flutter/lib/src/rendering/stack.dart

```diff
@@ -411,17 +401,17 @@ class RenderStack extends RenderBox
     }
   }
 
-  /// Whether overflowing children should be clipped. See [Overflow].
+  /// {@macro flutter.widgets.Clip}
   ///
-  /// Some children in a stack might overflow its box. When this flag is set to
-  /// [Overflow.clip], children cannot paint outside of the stack's box.
-  Overflow get overflow => _overflow;
-  Overflow _overflow;
-  set overflow(Overflow value) {
+  /// Defaults to [Clip.hardEdge], and must not be null.
```

Comment:
[nit] It might be useful to copy the "Some children in a stack might overflow its box." introduction before. New developers might not realize why Stack needs clipping. But this could be a rabbit's hole, talking the size of the stack versus positioned widgets. So: nit.